functions/icons.zsh

```diff
@@ -34,6 +34,7 @@ case $POWERLEVEL9K_MODE in
       TEST_ICON                      $'\uE891'              # 
       TODO_ICON                      $'\u2611'              # ☑
       BATTERY_ICON                   $'\uE894'              # 
+      HDD_ICON                       $'\uD83D\uDDB4'        # 🖴
```

Comment:
I'm using `awesome-patched` mode and cannot see this icon...
How about `$'\uE1AE'`?